app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -960,13 +960,20 @@ def rbac_group_right_tree(selected)
   def rbac_role_get_details(id)
     @edit = nil
     @record = @role = MiqUserRole.find_by_id(from_cid(id))
-    @role_features = @role.feature_identifiers.sort
-    @features_tree = rbac_build_features_tree
+    @rbac_menu_tree = build_rbac_feature_tree
   end
 
-  def rbac_build_features_tree
+  def build_rbac_feature_tree
     @role = @sb[:typ] == "copy" ? @record.dup : @record if @role.nil? # if on edit screen use @record
-    TreeBuilder.convert_bs_tree(OpsController::RbacTree.build(@role, @role_features, !@edit.nil?)).to_json
+
+    TreeBuilderOpsRbacFeatures.new(
```

Comment:
Just a small thing, but I'd put the methods on the first line and the hash keys into separate ones if RuboCop stays happy. It isn't that long code...